### PR TITLE
OSDOCS-1967: Add release note for network policy audit logging

### PR DESCRIPTION
**Pull request by jboxman**
- https://issues.redhat.com/browse/OSDOCS-1967

Preview: https://deploy-preview-33078--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes#ocp-4-8-network-policy-audit-logs

### CODE REVIEW

**Comment by jboxman** [comment]
@astoycos, first draft of release note for this; When a log is saved locally, is that to the node itself or inside a pod or some other concept of _local_?

I don't want to go into too much detail here, because it's a release note entry.

Thanks!

**Comment by jboxman** [comment]
@astoycos, how does this look? Thanks!

**Comment by astoycos** [comment]
This got lost in all my gh pings during this last week sorry about that 

> @astoycos, first draft of release note for this; When a log is saved locally, is that to the node itself or inside a pod or some other concept of local?

The log is saved locally to the node itself 

>The logs are in a syslog compatible format and can be saved locally, sent over a UDP connection, or directed to a UNIX domain socket. You can specify whether to log allowed connections, dropped connections, or allowed and dropped connections

Should be 

The logs are in a syslog compatible format and are be saved locally, but can also be sent over a UDP connection, or directed to a UNIX domain socket. You can specify whether to log allowed connections, dropped connections, or allowed and dropped connections

**Comment by jboxman** [comment]
@astoycos, updated.

Is the _node_ in this context one of the `ovnkube-node-XXXXX` nodes?

Thanks!

**Comment by jboxman** [comment]
@asood-rh, can you take a look at this when you have time? Thanks!

**Review by jeana-redhat — APPROVED** [review]
LGTM pending the xref you mentioned you're adding later :+1:

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -151,6 +151,12 @@ Packet data is not sent to the network flows collector. Packet-level metadata su
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/tracking-network-flows.adoc#tracking-network-flows[Tracking network flows].
 
+[id="ocp-4-8-network-policy-audit-logs"]
+==== Network policy audit logs
+
+If you use the OVN-Kubernetes cluster network provider, you can enable audit logging for network policies in a namespace. The logs are in a syslog compatible format and saved locally, but can also be sent over a UDP connection or directed to a UNIX domain socket. You can specify whether to log allowed connections, dropped connections, or allowed and dropped connections. For more information, see
```
Comment (asood-rh) [inline release_notes/ocp-4-8-release-notes.adoc:157]: @jboxman Could this be rephrased "You can specify whether to log allowed connections, dropped connections, or allowed and dropped connections" to "You can specify whether to log allowed connections and/or dropped connections by the network policy". or "You can specify whether to log allowed connections, dropped connections, or both allowed and dropped connections". Just a nit. Thanks!

**Pull request merged**